Orange/preprocess/impute.py

```diff
@@ -26,7 +28,65 @@ def transform(self, c):
         return numpy.where(numpy.isnan(c), self.value, c)
 
 
-class Average:
+class BaseImputeMethod:
+    name = ""
+    short_name = ""
+    description = ""
+    format = "{var.name} -> {short_name}"
```

Comment:
How about specifying this as `"{var.name} → {self.short_name}"`. Then `format_variable()` method below could call format it with `self.format.format(var=var, self=self)` and there would be no reason to override the method if `format` is already overriden?